docs/our_work/a_and_e_forecasting_tool.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Accident and Emergency (A&E) Forecasting Tool'
+summary: 'A probabilistic model which gives a three-week forecast for A&E departments, predicting their expected admissions loads'
+origin: ''
+tags: ['MODELLING', 'HOSPITAL', 'MONTE CARLO', 'MCMC', 'URGENT CARE', 'ACCIDENT AND EMERGENCY']
```

Comment:
check whether any of the currently existing tags sort of cover what you need here, instead of making new tags, but make new tags if absolutely needed

Reply:
trimmed this